src/main/java/seedu/trackermon/logic/commands/SortCommand.java

```diff
@@ -18,12 +18,12 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sort shows based on the input prefix. If there are "
             + "no prefix, it will sort by name in ascending order.\n"
-            + "if both prefixes for ascending and descending are used,\n"
+            + "if the value for ascending and descending is incorrect, \n"
             + "it will only sort by ascending.\n"
-            + "By default it sort by name then status,\n"
-            + "use so/ to sort by status then name.\n"
-            + "Parameters: [sna/] [snd/] [ssa/] [ssd/] [so/]...\n"
-            + "Example: " + COMMAND_WORD;
+            + "if all prefix are shown, by default it sort by name then status then rating then tags,\n"
```

Comment:
Should this be `all prefixes` instead? 
I would also rephrase
`by default it sort by name then status then rating then tags`  to
`by default, it sorts by name, then status, followed by rating, and finally tags`
just to avoid the repeating of then.
At the very least I think the usage of commas to separate the different sorting conditions should be included. Just a nitpick though.

Reply:
i think i should just remove the command message, it's not used